components/configs/rwn-optimizations/base/3-node/config-ingress.yaml

```diff
@@ -0,0 +1,12 @@
+---
+apiVersion: operator.openshift.io/v1
+kind: IngressController
```

Comment:
One could also use Router sharding here to affect the behavior of where the ingress sits, if it is not wanted on control plane hosts / supervisors and instead on the RWN: https://docs.openshift.com/container-platform/4.12/networking/ingress-operator.html#nw-ingress-sharding_configuring-ingress

Safe to say this is outside the scope for right now, but sharing nonetheless.

Reply:
Agree in that  out of scope. I can see use case ingress controller in edge worker node in real deployment assuming full cluster when RAN outside of OCP domain.